test/plugins/auto-load.spec.js

```diff
@@ -117,6 +117,38 @@ describe('AutoLoad plugin', () => {
     return skill.execute(event);
   });
 
+  it('should throw an error when no config file is provided', () => {
+    simple.mock(adapter, 'get', undefined);
```

Comment:
The `autoLoadAdapter` adapter has a valid get method. This test is to verify if the adapter users would pass through does not have a valid GET method, so we would throw an error for that